scripts/dev-db-reset.py

```diff
@@ -31,7 +36,14 @@ def populate():
 
 
 if __name__ == '__main__':
-    parse_args()
+    args = parse_args()
+    if args.keep:
+        from vimma.models import Permission
+        try:
+            Permission.objects.all()
```

Comment:
Let's revert the changes to this file and instead check if the DB exists in run.py – that seems the proper place for this.
This is turning from a straightforwrd “throw away and re-create the database” into “delete and recreate, or do nothing with -keep, except if the permissions are missing”.

As an aside, this file is just a helper for setting up dev, unused in production.